docs/blog/kustomize-installation.md

```diff
@@ -0,0 +1,200 @@
+---
+author: "Edvin 'NissesSenap' Norling"
+date: 2023-03-29
+title: "Install Grafana-operator using Flux and Kustomize"
+linkTitle: "Install Grafana-operator using Flux and Kustomize"
+description: "How to install grafana-operator using Flux and Kustomize"
+---
+
+As a part of grafana-operator v5.0.0-rc1 we introduce Kustomize as a way of installing the operator.
+
+To showcase this new feature, I thought why not use GitOps?
+
+GitOps is a rather well-used term nowadays, but you can summarize it in 4 steps.
+
+- Declarative
+- Versioned and immutable
+- Pulled automatically
+- Continuously reconciled
+
+To find out more about GitOps look at the CNCF GitOps working groups [documentation](https://opengitops.dev/).
+
+In this case I have decided to use [Flux](https://fluxcd.io/) to deploy grafana-opreator through GitOps, but there are other options.
+For example [ArgoCD](https://argo-cd.readthedocs.io/) which is also a graduated CNCF project just like Flux.
+
+This blog's focus is to showcase how you can use Kustomize to install the grafana-operator, I will take many shortcuts to keep this blog **simple**, read the official Flux documentation for best practices.
+
+## What we will do?
+
+We will install the grafana-operator using Flux and to manage our grafana instance and dashboard.
+
+Assuming that you follow the instructions of this blog, your Flux fleet-infra repository will look something like this.
+
+```.txt
+├── clusters
+│   └── my-cluster
+│       ├── flux-system
+│       │   ├── gotk-components.yaml
+│       │   ├── gotk-sync.yaml
+│       │   └── kustomization.yaml
+│       ├── grafana
+│       │   ├── dashboard.yaml
+│       │   ├── grafana.yaml
+│       │   └── kustomization.yaml
+│       ├── grafana-operator.yaml
+│       └── grafana.yaml
+```
+
+You can find all the files available to copy in the grafana-operator [repository](https://github.com/grafana-operator/grafana-operator/tree/master/docs/blog/flux-gitops),
+or you can just copy paste them from the blog.
+
+## Prerequisite
+
+In this example, I will use Kind as my cluster because I think it's simple, but the walkthrough should work with any Kubernetes solution.
```

Comment:
```suggestion
In this example, I will use [Kind](https://kind.sigs.k8s.io/docs/user/quick-start/) as my cluster because I think it's simple, but the walkthrough should work with any Kubernetes solution.
```